prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,10 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgress'];
+console.log(languages);
+const firstElement = languages[0];
+console.log(firstElement);
+const thirdElement = languages[2];
+console.log(thirdElement);
+const length = languages.length;
+console.log(length);
+const lastElement = languages[5];
```

Comment:
make sure you make use of the `length` variable when accessing the last index of the array

Reply:
@Zhiwutian Is this correct?
<img width="432" alt="Screenshot 2023-03-03 at 10 10 44 PM" src="https://user-images.githubusercontent.com/121325991/222879299-989ed609-b7ca-4a34-9f9c-a716d37b764c.png">